src/Nest/Mapping/Types/Geo/GeoShape/GeoOrientation.cs

```diff
@@ -1,16 +1,41 @@
-﻿using System.Runtime.Serialization;
-using Elasticsearch.Net;
-
+﻿using Elasticsearch.Net;
 
 namespace Nest
 {
-	[StringEnum]
+	[JsonFormatter(typeof(GeoOrientationConverter))]
 	public enum GeoOrientation
 	{
-		[EnumMember(Value = "cw")]
 		ClockWise,
-
-		[EnumMember(Value = "ccw")]
 		CounterClockWise
 	}
+
+	internal class GeoOrientationConverter : IJsonFormatter<GeoOrientation>
+	{
+		public void Serialize(ref JsonWriter writer, GeoOrientation value, IJsonFormatterResolver formatterResolver)
+		{
+			switch (value)
+			{
+				case GeoOrientation.ClockWise:
+					writer.WriteString("cw");
+					break;
+				case GeoOrientation.CounterClockWise:
+					writer.WriteString("ccw");
+					break;
+			}
+		}
+
+		public GeoOrientation Deserialize(ref JsonReader reader, IJsonFormatterResolver formatterResolver)
+		{
+			var enumString = reader.ReadString();
+			switch (enumString.ToLowerInvariant())
```

Comment:
Is `ToLowerInvariant()` necessary?

Reply:
Yes, ES capitalises.

Reply:
would it be better to check against captialized names, and avoid the `ToLowerInvariant()` call?

Reply:
...perhaps, but its not a very defensive approach.

Reply:
But it avoids the string allocation in a scenario where we know what to expect from the server, avoiding unneccessary work